.github/workflows/publish.yaml

```diff
@@ -0,0 +1,20 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registries:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Run docker build
+        run: make docker-build
```

Comment:
Maybe we want to keep the runs from build.yml in github and just pull them from there and push them to docker upon release?

Reply:
Unfortunately, GitHub Actions doesn't support direct communications between workflow. There is a workaround for that (store docker image as an artifact on flow 1 and then download it on flow 2) but it looks ugly for me

Reply:
I rather store the docker image than build it twice for no reason, it's also cheaper.